GUI/src/main/kotlin/ui/screens/DiffScreen.kt

```diff
@@ -46,9 +47,12 @@ fun DiffScreen(state: MutableState<AppState>) {
     val scope = rememberCoroutineScope()
     val navigator = FrameNavigation(state, scope)
     val showConfirmationDialog = remember { mutableStateOf(false) }
+    val frameGrabber = FrameGrabber(state)
+
     DisposableEffect(Unit) {
         onDispose {
             navigator.close()
+            frameGrabber.close()
```

Comment:
The line above closes the frameGrabber already. Or not, its closing another instance that lives within the navigator instance, right?

Reply:
I just removed the frameGrabber created in navigator as it serves no purpose anymore